src/common/OutputDataSocket.cc

```diff
@@ -386,10 +386,10 @@ void OutputDataSocket::append_output(bufferlist& bl)
 
   if (data_size + bl.length() > data_max_backlog) {
     ldout(m_cct, 20) << "dropping data output, max backlog reached" << dendl;
+    return;
```

Comment:
as discussed, this warning message should be more visible if we're actually dropping data

Reply:
agreed